drivers/vulkan/vulkan_context.cpp

```diff
@@ -101,6 +101,7 @@ VkResult VulkanContext::vkCreateRenderPass2KHR(VkDevice p_device, const VkRender
 			attachments.push_back(att);
 		}
 
+		Vector<Vector<VkAttachmentReference>> attachment_references;
```

Comment:
Wouldn't it be more clear to define the attachments here (but not initialize them) like:
```
Vector<VkAttachmentReference> input_attachments;
Vector<VkAttachmentReference> color_attachments;
Vector<VkAttachmentReference> resolve_attachments;
Vector<VkAttachmentReference> depth_attachments;
```

Reply:
Im not sure I understand the suggestion.
There are vectors of attachments created in each subpass, so there would still need to be a vector of vectors defined here.

Reply:
Ah, you are right. I was thinking we only had a total of 4 ``Vector<VkAttachmentReference>``, but we actually have 4 for each subpass count. My suggestion would result in only the last subpass staying in scope